### DISCUSSION TITLE
Please re-enable the legacy UDO debugger while the Analytics Debugger is awaiting the functionality

### DISCUSSION THREAD

**Discussion by cgullestrup** [analytics-debugger/analytics-debugger-browser-extension#30] [Q&A] [answered]
Hi @thyngster,

We just woke up to a non-functioning UDO debugger, telling us that the functionality has been incorporated into the Analytics Debugger. But it seems it is not yet the case:
https://www.thyngster.com/analytics-debugger-2-0-0-chrome-extension-release#:~:text=Adobe%20Target-,Tealium%20IQ,-Yandex%20Metrika

Can you please re-enable the legacy UDO debugger, until the features have been fully migrated?

![image](https://user-images.githubusercontent.com/73661542/232716804-350145e3-9a02-42ee-8ffc-d4fbfc8b73d4.png)

Thanks in advance :-)

**Comment by thyngster** [comment]
I apologize for the inconvenience caused by CWS publishing. I initially sent the Analytics Debugger update and then 24 hours later, I sent the UDO Debugger update. However, Google only updated the latter despite it being uploaded later.

Unfortunately, I cannot currently take any action to resolve this issue. It may take up to three weeks to send a new update, which would only prolong the situation. Would you be willing to wait a bit longer for the update to be applied?

I apologize sincerely for this predicament, but unfortunately, my hands are tied. I have opened a ticket with Google to investigate what has caused this issue.

**Reply by cgullestrup** [reply]
Hi David  - thanks for getting back to me! And thanks for all your great work :-)

I know it is out of your hands - I was unable to see any new code here on GitHub indicating that there was a new version of the extension.

Are you saying that you recently updated the Analytics Debugger as well to support Tealium? If that is the case, then we just have to be patient and await its release in the Google Extension/Web store :-)

**Comment by thyngster** [comment, marked as answer]
I had 4 extensions ( Tealium, Adobe Stack, Google Stack, and Yandex ). Keeping things up to date was not easy, so I merged all them on Analytics Debugger, plus I'm adding support for other tools like Matomo/Piwik Pro/Amplitude/etc. 

This will help on being able to offer more updates and just keeping one extension. So when it's up the version 2.3.0 for analytics debuger, just enable the Tealium IQ, and you'll be able ot keep working as before. Some updates will be coming to the Tealium debugging, including getting back some features that stopped working over the years.

**Reply by cgullestrup** [reply]
Great, thanks for your help! We will patiently await the release of 2.30 :-)

Have a great day!

**Reply by cgullestrup** [reply]
I can confirm that the new version, 2.3.0, of the Analytics Debugger is available in the Chrome Web Store. 

Thank you again for the help - it looks really good, and I really like the addition of the Adobe Analytics debugger :-)
